test/jobs/JobsManager.js

```diff
@@ -284,6 +287,15 @@ contract("JobsManager", accounts => {
             await expectThrow(jobsManager.verify(jobId, segmentSequenceNumber, utils.bufferToHex(d0), utils.bufferToHex(tD0), utils.bufferToHex(bSig0), proof, {from: accounts[0]}))
         })
 
+        it("should throw if segment not signed by broadcaster", async () => {
```

Comment:
A test for `verify` in which the broadcaster signature is incorrect was previously missing, so added it in here